index.ts

```diff
@@ -18,6 +18,21 @@ interface ZM {
 
 export type { Position };
 
+/**
+ * User-facing options for the WKT stringifier.
+ */
+export interface WktStringifyOptions {
+  /**
+   * Version of the SFA standard to follow:
+   * https://www.ogc.org/standards/sfa/
+   *
+   * 1.2.1 has the difference of including parenthesis around
+   * MULTIPOINT strings: MULTIPOINT((X Y),...)
+   * instead of MULTIPOINT(X Y, ...)
+   */
+  version?: "1.1.0" | "1.2.1";
```

Comment:
https://www.ogc.org/standards/sfa/

Technically multipoint comes from 1.2.0
```suggestion
  version?: "1.1.0" | "1.2.0";
```